src/components/card/card-dialog/CardDialog.js

```diff
@@ -37,6 +41,19 @@ function CardDialog({ addCard }) {
     clearState();
   };
 
+  const editCard = () => {
+    const isInvalid = isBlank(cardTitle) || isBlank(cardBody);
+    if (!isInvalid) {
+      updCard({
+        id: cardId,
+        title: cardTitle.trim(),
+        body: cardBody.trim(),
+      });
+    }
+    setOpen((prev) => !prev);
+    clearState();
+  };
+
```

Comment:
Seems that I've almost repeated saveCard method. How to do it better, cause difference is in 2 methods: updCard() or addCard().

Reply:
Fixed